### PR TITLE
Allow users to specify (via conf.py) the path to save log files.  Issue #303

### PR DESCRIPTION
**Pull request by vladimir-v-diaz**
Initial implementation for issue #303

### CODE REVIEW

**Comment by SantiagoTorres** [comment]
Nice :+1:

File: tuf/log.py [hunk 1]
```diff
@@ -104,18 +105,25 @@
...
+  file_handler.setLevel(_DEFAULT_FILE_LOG_LEVEL)
+  file_handler.setFormatter(formatter)
+else:
+  pass
+
 
 # Set the logger and its settings.
 logger = logging.getLogger('tuf')
 logger.setLevel(_DEFAULT_LOG_LEVEL)
-logger.addHandler(file_handler)
```
Comment (SantiagoTorres) [inline tuf/log.py]: I'm wondering if this, and the previous if can be merged together for simplicity. They check for the same condition.

Comment (vladimir-v-diaz) [inline tuf/log.py]: I now set the logger (prior to adding a potential file handler) to avoid these multiple if-else statements.

**Comment by davedoesdev** [comment]
Yes, that's great - thanks

**Pull request merged**